data/json/items/armor.json

```
@@ -3079,7 +3079,7 @@
     "//": "DDA monocle is optical-grade.  Costume ones go for about $10.",
     "description": "An essential article of the gentleman's apparel.  Also corrects near-sightedness.  The concave lenses diffuse the sunlight, rendering them useless for starting fires.",
     "weight": 16,
-    "volume": 0,
+    "volume": "0 L",
```

```suggestion
    "volume": "5 ml",
```
Going off of https://www.nearsights.com/pages/sizing with a width of 4mm.